### PR TITLE
Add script to create custom quality profile

### PR DESCRIPTION
**Pull request by Silicoman**
Hi,

Reuse script shell to develop a new feature **"creation custom ecocode quality profile"**.
The script is not clean like other scripts shell in the folder but it does the job ! 

Execute script `install_profile.sh` to got new quality *ecocodeProfile* profile forked from "SonarWay". You will have a new profile for each language.
Ecocode profile will got new rules from plugins tags by "eco-conception" 

Details have be explained inside the readme.md

This script will simplify user experience  like #21.

### CODE REVIEW

File: tools_init/_core.sh
Comment (dedece35) [inline tools_init/_core.sh:35]: please change this comment : 
- $1 is not used in method
- make a desctiption of method purpose

File: tools_init/_core.sh
Comment (dedece35) [inline tools_init/_core.sh:84]: wrong description : creation and not a find

File: tools_init/_core.sh
Comment (dedece35) [inline tools_init/_core.sh:101]: tags list seperated with a comma ?

**Review by dedece35 — CHANGES_REQUESTED** [review]
HI @Silicoman 

review done. please check my comments.
thank you.

**Comment by Silicoman** [comment]
Hi @dedece35 
I have done corrections. 

Happy Holidays!

File: tools_init/install_profile.sh
Comment (dedece35) [inline tools_init/install_profile.sh:22]: I think adding some logs to steps will help the user. For example :
- `echo -e "Creating profile : $PROFILE_ECOCONCEPTION"` before `create_profile_sonarapi` call
- `echo -e "Changing parent profile"` before `change_parent_profile_sonarapi` call
- `echo -e "Activating ecocode rules with tag '$TAG_ECOCONCEPTION' for key_profile '$key_profile'"` before `activate_rules_ecocode_profile_sonarapi` call

I did it locally to understand my local execution to check.

File: tools_init/readme.md
Comment (dedece35) [inline tools_init/readme.md:31]: I think you would tell that a SonarQube token with "User" rights is mandatory (because with "general" rights will produce authorization errors)

File: tools_init/readme.md
Comment (dedece35) [inline tools_init/readme.md:51]: I would say "activate existing rules associated with mentionned tag"

File: tools_init/readme.md
Comment (dedece35) [inline tools_init/readme.md:74]: please see my review note above

**Review by dedece35 — CHANGES_REQUESTED** [review]
@Silicoman thank you for modifications, but after testing it locally, I have some review notes again. sorry :p

File: tools_init/readme.md
Comment (Silicoman) [reply tools_init/readme.md:31]: I  was thinking permission is a global problematic of this package scripts (tag + profile).
 Needed  to put   ' admininister Quality Profiles' to update tag and profile at the top of the readme.

Comment (dedece35) [reply tools_init/readme.md:31]: ok with you. can you add it in documentation ? thank you

**Review by dedece35 — APPROVED** [review]
Hi @Silicoman,
ok with your PR.
you can merge the PR with squash option and a small git comment please.

**Comment by Silicoman** [comment]
> Hi @Silicoman, ok with your PR. you can merge the PR with squash option and a small git comment please.
Normally, it was one option to select before clicking on merge button.

**Pull request merged**
